src/auth/AuthContext.tsx

```diff
@@ -101,6 +101,10 @@ const getAuthenticationType = (
 
 export type ConfirmationErrorCode = 'invalid_code' | 'unknown_error';
 export type PhoneSignInErrorCode = 'invalid_phone' | 'unknown_error';
+export type VippsSignInErrorCode =
+  | 'access_denied'
+  | 'outdated_app_version'
+  | 'unknown_error';
```

Comment:
Don't think we have many of the error states that make sense to the user. So it's unknown_error most of the time.